### DISCUSSION TITLE
Can't view sessions

### DISCUSSION THREAD

**Discussion by ASlava12** [tmux-python/libtmux#380] [Q&A] [answered]
The launch happens inside the docker container:
```
>>> import libtmux
>>> s = libtmux.Server()
>>> s.list_sessions()
```

Outside the container, everything works. There is a tmux inside the container.

**Comment by tony** [comment]
@ASlava12 

What output does it give? Anything at all?

Possibly related to #265

**Comment by tony** [comment, marked as answer]
Does setting `LANG` help? re: https://github.com/tmux-python/libtmux/issues/265#issuecomment-639915895

**Comment by ASlava12** [comment]
Hello!

The problem was really related to the docker environment, the problem was solved by the link

**Comment by tony** [comment]
Good to know this was resolved @ASlava12!
